### PR TITLE
[Flex Counters] Delay flex counters even if tables are present in the DB

### PR DESCRIPTION
**Pull request by shlomibitton**
**What I did**
Check if delay indicator flag is exist and 'true', if it does, skip the counter enablement.

**Why I did it**
Currently if flex counters tables are present in config DB the delay mechanism will not take place.
This change is to make sure the delay will take place even if the tables are present in the DB.

**How I verified it**
Observer counters are created after enable_counters script is called.

**Details if related**

### CODE REVIEW

File: orchagent/flexcounterorch.cpp [hunk 1]
```diff
@@ -144,6 +149,8 @@ void FlexCounterOrch::doTask(Consumer &consumer)
                     fieldValues.emplace_back(FLEX_COUNTER_STATUS_FIELD, value);
                     m_flexCounterGroupTable->set(flexCounterGroupMap[key], fieldValues);
                 }
+                else if(field == FLEX_COUNTER_DELAY_STATUS_FIELD)
+                {}
```
Comment (kcudnik) [inline orchagent/flexcounterorch.cpp:153]: add at least comment why this is ignored?

File: orchagent/flexcounterorch.cpp [hunk 2]
```diff
@@ -86,6 +86,11 @@ void FlexCounterOrch::doTask(Consumer &consumer)
 
         if (op == SET_COMMAND)
         {
+            auto it = std::find(std::begin(data), std::end(data), FieldValueTuple(FLEX_COUNTER_DELAY_STATUS_FIELD, "true"));
+            if (it != data.end())
```
Comment (kcudnik) [inline orchagent/flexcounterorch.cpp:90]: maybe empty line before if

Comment (shlomibitton) [reply orchagent/flexcounterorch.cpp:91]: Done.

File: orchagent/flexcounterorch.cpp (hunk 1 above)
Comment (shlomibitton) [reply orchagent/flexcounterorch.cpp:157]: Done.

**Comment by liat-grozovik** [comment]
@shlomibitton is it relevant for 202106?
@vaibhavhd can you pls help to review?

**Comment by liat-grozovik** [comment]
@shlomibitton pls handle LGTM errors

**Comment by shlomibitton** [comment]
@liat-grozovik Yes it is.

**Comment by shlomibitton** [comment]
@kcudnik LGTM is still failing since FLEX_COUNTER_DELAY_STATUS_FIELD is not declared even though PR https://github.com/Azure/sonic-swss-common/pull/523 already merged, this field should be declared.
Do you know how can we fix it?

**Comment by kcudnik** [comment]
it was merged on master ? in swss-common?

**Comment by shlomibitton** [comment]
yes
https://github.com/Azure/sonic-swss-common/pull/523
https://github.com/Azure/sonic-buildimage/pull/8565

**Review by vaibhavhd — DISMISSED** [review]

File: orchagent/flexcounterorch.cpp (hunk 1 above)
Comment (kcudnik) [reply orchagent/flexcounterorch.cpp:157]: i more was thinking about commend in IF itself where ou have {}, 
```
{
 // comment
}
```

Comment (shlomibitton) [reply orchagent/flexcounterorch.cpp]: how about now?

**Review by kcudnik — APPROVED** [review]

**Pull request merged**